presto-docs/src/main/sphinx/release/release-0.206.rst

```diff
@@ -0,0 +1,56 @@
+=============
+Release 0.206
+=============
+
+General Changes
+---------------
+
+* Fix query processing error for certain queries when ``dictionary_aggregation`` is enabled.
+* Fix query plan for aggregations over a single group,
+  e.g. ``select c1, sum(c2) from t where c1 = 'foo' group by c1``
+* Fix average row size computation to account for non-null rows only
+* Fix high memory pressure on the coordinator during execution for queries
+  using bucketed execution.
+* Add new semantics for date time types. The new semantics conforms to SQL standard.
+  It affects ``TIMESTAMP`` (aka ``TIMESTAMP WITHOUT TIME ZONE``) type,
+  ``TIME`` (aka ``TIME WITHOUT TIME ZONE``) type, and ``TIME WITH TIME ZONE`` type.
+
+.. note::
+
+  The legacy behavior remains default.
+  At this time, it is not recommended to turn on the new semantics.
+  For any connectors with support of date time types, code change is required before the connector
+  can work correctly with the new semantics. None of the bundled connectors have been updated yet.
+  In addition, the new semantics is also not stable yet, as breaking changes are planned.
+
+* Add :func:`ST_Union`, :func:`ST_Geometries`, :func:`ST_PointN`, :func:`ST_InteriorRings`,
+  :func:`ST_InteriorRingN` geospatial functions.
+* Add :func:`split_to_multimap` function.
+* Expand :func:`approx_distinct` function to support the following types:
+  ``INTEGER``, ``SMALLINT``, ``TINYINT``, ``DECIMAL``, ``REAL``, ``DATE``,
+  ``TIMESTAMP``, ``TIMESTAMP WITH TIME ZONE``, ``TIME``, ``TIME WITH TIME ZONE``, ``IPADDRESS``.
+* Add the resource group id column in the ``system.runtime.queries`` table.
+* (@bowdencm  @nezihyigitbasi: should we have this as it's still WIP and not usable at all?)
+  Add SET PATH statement and CURRENT_PATH expression.
+  Note only parser support is available. Path-based function resolution is not yet implemented.
+* Add support for executing ORDER BY without LIMIT in a distributed manner.
+* (@sopel39: What is the user impact?)
+  Improve data size estimation heuristics to account for auxiliary objects overhead.
```

Comment:
`auxiliary objects overhead` -> `for the overhead of temporary objects`?
Also where are these (in which part of the system) size estimation heuristics, this is a pretty generic statement?

Reply:
it's not about temporary objects -- it's about nullity arrays (or offset arrays in case of varchar blocks). They take up memory the very same way actual data does and this wasn't considered.

> Also where are these (in which part of the system) size estimation heuristics, this is a pretty generic statement?

This is generic, in stats/cost computations (precisely, the `PlanNodeStatsEstimate` class)